jsf/injection/src/main/java/org/jboss/as/jsf/injection/AnnotationMap.java

```diff
@@ -60,7 +60,7 @@
     static {
         // These classes need to be loaded in order!  Some can't be loaded if the Jakarta Server Faces version is too old.
 
-        try { // all of the following classes are available from JSF 2.0 and JSF 2.1
+        try { // all of the following classes are available from Faces 2.0 and Faces 2.1
```

Comment:
@fjuma  do you agree that these references to `JSF/Faces 2.0/2.1` should change to `Jakarta Server Faces 2.3`?